install.md

```diff
@@ -1,22 +1,22 @@
 Installation
 ===================
 
-To get started with OpenCost and Kubecost, **the recommended path is to [install Kubecost community version](http://kubecost.com/install)**. This install method is available for free and leverages the Kubecost helm chart. It provides access to all OpenCost and Community Kubecost functionality and can scale to large clusters. This will also provide a token for trialing and retaining data across different Kubecost product tiers.
+To get started with OpenCost and Kubecost, **the recommended path is to [install Kubecost community version](http://kubecost.com/install)**. This installation method is available for free and leverages the Kubecost Helm Chart. It provides access to all OpenCost and Kubecost community functionality and can scale to large clusters. This will also provide a token for trialing and retaining data across different Kubecost product tiers.
```

Comment:
Note/TODO: let's standardize how we talk about Community Kubecost

Reply:
Correct, I can add that to the style guide. I can't find any documentation for a "Community Kubecost" outside of this page. Is there any example of this being a proper noun? Other examples of community in the docs use it in a non-specific casual way.